Sources/Widgets/Widgets3D/LineWidget/behavior.js

```diff
@@ -30,85 +27,240 @@ export default function widgetBehavior(publicAPI, model) {
+      callData.position.y,
+      worldMousePos[2]
+    );
+    return mousePos;
+  }
+
+  publicAPI.isHandleGlyph2D = (handleShape) =>
+    handleShape === HandleRepresentationType.ARROWHEAD3 ||
+    handleShape === HandleRepresentationType.ARROWHEAD4 ||
+    handleShape === HandleRepresentationType.ARROWHEAD6 ||
+    handleShape === HandleRepresentationType.STAR ||
+    handleShape === HandleRepresentationType.CIRCLE ||
+    handleShape === HandleRepresentationType.VIEWFINDER ||
+    handleShape === HandleRepresentationType.DISK;
+
+  // Handle orientation & rotation ---------------------------------------------------------
+
+  function orientFirstHandleBeforeSecondHandlePlacement(callData) {
+    const handle1Origin = handle1.getOrigin();
+    const mousePos = computeMousePosition(handle1Origin, callData);
+    return computeDirectionWithTwoPoints(handle1Origin, mousePos);
+  }
+
+  function orientHandlesWithCompleteWidget(nb) {
+    let dir = [0, 0, 0];
+    const handle1Origin = handle1.getOrigin();
+    const handle2Origin =
+      nb !== 3
+        ? handle2.getOrigin()
+        : model.widgetState.getMoveHandle().getOrigin();
+    dir = computeDirectionWithTwoPoints(handle1Origin, handle2Origin);
+    if (nb === 2 || nb === 3) {
+      vtkMath.multiplyScalar(dir, -1);
+    }
+    return dir;
+  }
+
+  function orientHandle(nb, direction) {
+    if (getNbHandles(model) !== 2) {
+      return;
     }
-    if (isHandleOrientable(model.handle2Shape)) {
-      updateHandleDirection(HandleBehavior.HANDLE2);
+    console.log(getNbHandles(model));
+
+    if (
+      (nb === 1 && handle1.getShape() === 'cone') ||
+      ((nb === 2 || nb === 3) && handle2.getShape() === 'cone')
+    ) {
+      model.representations[nb - 1].getGlyph().setDirection(direction);
+    } else {
+      model.representations[nb - 1].setOrientation(direction);
+    }
+  }
+
+  function updateHandleDirection(nb) {
+    const direction = orientHandlesWithCompleteWidget(nb);
+    orientHandle(nb, direction);
+  }
+
+  publicAPI.updateHandleDirections = () => {
+    if (isHandleOrientable(handle1.getShape())) {
+      updateHandleDirection(1);
+    }
+    if (isHandleOrientable(handle2.getShape())) {
+      updateHandleDirection(2);
+    }
+  };
+
+  publicAPI.rotateHandlesToFaceCamera = () => {
+    if (
+      (model.representations[0].getFaceCamera() === FaceCameraStates.NONE &&
+        publicAPI.isHandleGlyph2D(handle1.getShape())) ||
+      model.representations[0].getFaceCamera() === FaceCameraStates.TRUE
+    ) {
+      model.representations[0].setViewMatrix(
+        Array.from(model.camera.getViewMatrix())
+      );
+    }
+    if (
+      (model.representations[1].getFaceCamera() === FaceCameraStates.NONE &&
+        publicAPI.isHandleGlyph2D(handle2.getShape())) ||
+      model.representations[1].getFaceCamera() === FaceCameraStates.TRUE
+    ) {
+      model.representations[1].setViewMatrix(
+        Array.from(model.camera.getViewMatrix())
+      );
+    }
```

Comment:
I wouldn't bother doing the test here (wether FaceCamera is forced or not), I would let the representation deal with it.

Here you simply always set the view matrix.